README.md

```diff
@@ -11,5 +11,8 @@ Auto-complete system using Neo4j graph database for storing data and providing f
 * Serialization and deserialization of servers for data exchange.
 
 ### How to use
+* Run `virtualenv -p python3 env`
```

Comment:
we can just do this:
`
# macOS/Linux
sudo apt-get install python3-venv    # If needed
python3 -m venv env

# Windows
python -m venv env
`